index/index.go

```diff
@@ -929,8 +929,8 @@ func (r *Reader) Postings(name, value string) (Postings, error) {
 
 // SortedPostings returns the given postings list reordered so that the backing series
 // are sorted.
-func (r *Reader) SortedPostings(p Postings) Postings {
-	return p
+func (r *Reader) SortedPostings(post Postings) Postings {
+	return post
```

Comment:
when we use a variable in the next 1-2 lines it is fine to have it as a single letter.

Reply:
Ok, will change it.